templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -151,10 +305,11 @@ Resources:
       StackId:
         Fn::ImportValue: !Ref OpsWorksStackID
 
-  OpswxEC2Web1:
+  OpswxEC2Web1{% if item.tags["PowerManaged"] == "Yes" %}PowerManaged{% endif %}:
```

Comment:
_"{% if item.tags["PowerManaged"] == "Yes" %}PowerManaged{% endif %}"_ is used 16 times,

I wonder if it would be worthwhile for the stack and hostnames be made into variables to improve readability?

Reply:
since its in an if loop, and if loops are very nasty in trying to alter values, i'd try and not touch it if possible.